CONTRIBUTING.md

```diff
@@ -12,17 +12,20 @@
     of the following release. If adding an internal function `_foo`, define it
     in __src/internal/_foo.js__.
 
-4.  Make one or more atomic commits. Each commit should have a descriptive
+4.  If adding new functions or removing existing ones, consider to update
+    `./index.js` exports list with corresponding changes.
```

Comment:
This could be included in step 3:

    If adding a function `R.foo`, define it in __src/foo.js__, require it in __index.js__, and include unit tests in __test/foo.js__.
                                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

We needn't document the process for *removing* functions as this is only ever done by members of the core team.